server/db/seedpsql.js

```diff
@@ -0,0 +1,38 @@
+const faker = require('faker');
+const dbpsql = require('./indexpsql.js');
+
+const tenMil = 10000000;
+
+const randomInt = function (max) {
+  let num = Math.ceil(Math.random() * max);
+  // all IDs start at 1 and not 0
+  if (num < 1) {
+    num = 1;
+  }
```

Comment:
Is this conditional necessary? Wouldn't Math.ceil ensure that the lowest generated number is 1?

Reply:
I think you're right, thanks :)